packages/rds/initial-bootstrap.ts

```diff
@@ -95,6 +98,13 @@ async function handleDelete(connectConfig: Config): Promise<void> {
   await connection.query(`DROP DATABASE IF EXISTS ${connectConfig.databaseName} WITH FORCE`);
 }
 
+async function migrate(connectConfig: Config): Promise<void> {
+  const connection = await connect(connectConfig);
+  const migrations = await connection.runMigrations({ transaction: "none" });
+  await connection.close();
+  migrations.forEach((migration) => console.log(migration));
```

Comment:
Looks like this method should only return migrations that we actually ran, although TypeORM also uses its own internal logger to log each one as well.

Reply:
Yeah, my plan was to remove this before going non-draft.